data/generateData.js

```diff
@@ -0,0 +1,93 @@
+const faker = require('faker');
+const Chance = require('chance');
+const fs = require('fs');
+const stream = require('stream');
+
+const chance = new Chance();
+
+// to create files to put in db
+const createFile = (streamWriter, data, entries) => {
+  let i = entries;
+  const write = () => {
+    let drained = true;
+    do {
+      i -= 1;
+      drained = streamWriter.write(data());
+    } while (i > 0 && drained);
+    if (i > 0) {
+      streamWriter.once('drain', write);
+    }
+  };
+  write();
+};
+
+// create file for restaurants table
+const restaurantStream = fs.createWriteStream('./data/files/restaurants.csv');
```

Comment:
This is a bit style-istic/personal choice so if you find the way you're doing it better for you then stick with it but the paths to the files here are pretty clear in what the data contained within them is, so personally I'd rather opt to just fold the `fs.createWriteStream()` call directly into the `createFile()` call like `createFile(fs.createWriteStream('./data/files/restaurants.csv'), generateRestaurantData, 10000000)` rather than setting up these variables that just all do the same thing and are discarded immediately afterwards.